gslib/commands/hmac.py

```diff
@@ -241,6 +318,28 @@ class HmacCommand(Command):
           'update': _update_help_text,
       })
 
+  gcloud_storage_map = GcloudStorageMap(
```

Comment:
Instead of putting this here, maybe just declare it in the `else` block of `get_gcloud_storage_args`

Reply:
Done.